lib/components/Box/Box.docs.tsx

```diff
@@ -392,7 +392,7 @@ const docs: ComponentDocs = {
               These background colours are likely to change over time, so it’s
               important that you only use them within the semantic context they
               were designed for, e.g. only using the “selection” colour for
-              actual user selections. If you choose colours based soley on their
+              actual user selections. If you choose colours based solely on their
               appearance, your application’s colours may change in unexpected
               ways when upgrading Braid.
```

Comment:
Fix prettier manually 🤞 
```suggestion
              actual user selections. If you choose colours based solely on
              their appearance, your application’s colours may change in
              unexpected ways when upgrading Braid.
```